### PR TITLE
Updated some graphs on the bpmn explorer dashboard 

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js [hunk 1]
```diff
@@ -455,4 +455,46 @@ function setDatePicker (dateElement) {
...
+    var x = document.getElementById("processInstanceCount").value;
+    var url = httpUrl + "/" + CONTEXT + "/stats?update=true&option=processinstancecount&id=" + x ;
+
+    $.ajax({
+        type: 'GET',
+        contentType: "application/json",
+        url: url,
+        success: function (data) {
+
+            var array = eval('('+data+')');
```
Comment (amalhub) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: Spaces missing between operators. (Apply code formatting)

**Comment by NatashaWso2** [comment]
Fixed with commit 6401754

File: components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag [hunk 2]
```diff
@@ -43,9 +43,9 @@
 			else if( option === "avgprocessduration"){
 				response.content = processBarChartAvgTime;
 			}
-			else{
-
-			}
+			else if(option === "processinstancecount"){
```
Comment (cnapagoda) [inline components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag]: Formatting issue

File: components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag [hunk 3]
```diff
@@ -43,9 +43,9 @@
 			else if( option === "avgprocessduration"){
 				response.content = processBarChartAvgTime;
 			}
-			else{
```
Comment (cnapagoda) [inline components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag]: Formatting issue

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js [hunk 4]
```diff
@@ -455,4 +455,46 @@ function setDatePicker (dateElement) {
...
+            google.load("visualization", "1", {packages:["corechart"]});
+            google.setOnLoadCallback(drawChart(array));
+
+
+            function drawChart(data) {
+                var dataArr = [['Process Name', 'Instance Count']];
+                for(var i = 0;i < data.length;i++){
```
Comment (cnapagoda) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: Formatting issue

File: components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag (hunk 2 above)
Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag]: Already fixed and merged
https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L656

File: components/bpmn/bpmn-explorer-web/src/web/js/actions.js (hunk 4 above)
Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/js/actions.js]: Already fixed and merged
https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L656

File: components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag (hunk 2 above)
Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag]: This code snippet has been moved to reportController.jag and the formatting issue is fixed there.
Fixed with commit 130406a

File: components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag (hunk 3 above)
Comment (NatashaWso2) [inline components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag]: This code snippet has been moved to reportController.jag and the formatting issue is fixed there.
Fixed with commit 130406a

**Pull request merged**